Intermittent crash in `copy_ranges` test on dx12 in CI

This failure is from https://github.com/gfx-rs/wgpu/actions/runs/17027857707/job/48265653345?pr=8085:

```
[pass] webgpu:api,validation,encoding,cmds,copyTextureToTexture:copy_ranges:dimension="3d";copyBoxOffsets={"x":1,"y":0,"z":0,"width":-1,"height":0,"depthOrArrayLayers":-2};srcCopyLevel=0;dstCopyLevel=0 (16ms). Log:
[pass] webgpu:api,validation,encoding,cmds,copyTextureToTexture:copy_ranges:dimension="3d";copyBoxOffsets={"x":1,"y":0,"z":0,"width":-1,"height":0,"depthOrArrayLayers":-2};srcCopyLevel=0;dstCopyLevel=1 (16ms). Log:
error: process didn't exit successfully: `D:\a\wgpu\wgpu\target\llvm-cov-target\debug\cts_runner.exe ./tools/run_deno --verbose 'webgpu:api,validation,encoding,cmds,copyTextureToTexture:*'` (exit code: 2173)
error: process didn't exit successfully: `'C:\Users\runneradmin\.rustup\toolchains\1.88-x86_64-pc-windows-msvc\bin\cargo.exe' run --manifest-path 'D:\a\wgpu\wgpu\Cargo.toml' --target-dir 'D:\a\wgpu\wgpu\target\llvm-cov-target' -p cts_runner --bin cts_runner -- ./tools/run_deno --verbose 'webgpu:api,validation,encoding,cmds,copyTextureToTexture:*'` (exit code: 2173)
Error: CTS failed

Caused by:
    command exited with non-zero code `cargo llvm-cov --no-cfg-coverage --no-report run --manifest-path D:\a\wgpu\wgpu\Cargo.toml -p cts_runner --bin cts_runner -- ./tools/run_deno --verbose webgpu:api,validation,encoding,cmds,copyTextureToTexture:*`: 1
error: process didn't exit successfully: `target\debug\wgpu-xtask.exe cts --llvm-cov --backend dx12` (exit code: 1)
Error: Process completed with exit code 1.
```

I doubt that it is related to the change in that PR. There is also a bug for dx12-only crashes of this test in Firefox: https://bugzilla.mozilla.org/show_bug.cgi?id=1982374

--- a/cts_runner/fail.lst
+++ b/cts_runner/fail.lst
@@ -23,7 +23,6 @@ webgpu:api,validation,createBindGroup:buffer,resource_state:* // 0%, https://git
 webgpu:api,validation,createBindGroup:external_texture,* // 0%, no external external texture in deno
 webgpu:api,validation,createBindGroup:texture,resource_state:* // crash, https://github.com/gfx-rs/wgpu/issues/7881
 webgpu:api,validation,createView:texture_state:* // 0%, https://github.com/gfx-rs/wgpu/issues/7881
-webgpu:api,validation,encoding,cmds,copyTextureToTexture:copy_ranges:* // ***, https://github.com/gfx-rs/wgpu/issues/8118
 webgpu:api,validation,encoding,cmds,debug:* // 92%, https://github.com/gfx-rs/wgpu/issues/8039
 webgpu:api,validation,encoding,cmds,render,draw:max_draw_count:* // https://github.com/gfx-rs/wgpu/issues/8737
 webgpu:api,validation,encoding,cmds,render,indirect_draw:indirect_offset_alignment:* // render bundle

--- a/cts_runner/test.lst
+++ b/cts_runner/test.lst
@@ -133,19 +133,7 @@ webgpu:api,validation,encoding,beginRenderPass:*
 webgpu:api,validation,encoding,cmds,clearBuffer:*
 webgpu:api,validation,encoding,cmds,compute_pass:*
 webgpu:api,validation,encoding,cmds,copyBufferToBuffer:*
-webgpu:api,validation,encoding,cmds,copyTextureToTexture:copy_aspects:*
-webgpu:api,validation,encoding,cmds,copyTextureToTexture:copy_ranges_with_compressed_texture_formats:*
-// Intermittently fails on dx12, https://github.com/gfx-rs/wgpu/issues/8118
-fails-if(dx12) webgpu:api,validation,encoding,cmds,copyTextureToTexture:copy_ranges:*
-webgpu:api,validation,encoding,cmds,copyTextureToTexture:copy_with_invalid_or_destroyed_texture:*
-webgpu:api,validation,encoding,cmds,copyTextureToTexture:copy_within_same_texture:*
-webgpu:api,validation,encoding,cmds,copyTextureToTexture:depth_stencil_copy_restrictions:*
-webgpu:api,validation,encoding,cmds,copyTextureToTexture:mipmap_level:*
-webgpu:api,validation,encoding,cmds,copyTextureToTexture:multisampled_copy_restrictions:*
-webgpu:api,validation,encoding,cmds,copyTextureToTexture:sample_count:*
-webgpu:api,validation,encoding,cmds,copyTextureToTexture:texture_format_compatibility:*
-webgpu:api,validation,encoding,cmds,copyTextureToTexture:texture_usage:*
-webgpu:api,validation,encoding,cmds,copyTextureToTexture:texture,device_mismatch:*
+webgpu:api,validation,encoding,cmds,copyTextureToTexture:*
 webgpu:api,validation,encoding,cmds,debug:debug_group_balanced:encoderType="compute%20pass"
 webgpu:api,validation,encoding,cmds,debug:debug_group_balanced:encoderType="non-pass"
 //FAIL: webgpu:api,validation,encoding,cmds,debug:debug_group_balanced:encoderType="render%20bundle"

